site/sitemap.xml

```diff
@@ -1061,35 +1061,35 @@
   <changefreq>weekly</changefreq>
 </url>
 <url>
-  <loc>https://spark.apache.org/spark-connect/</loc>
+  <loc>https://spark.apache.org/pandas-on-spark/</loc>
```

Comment:
Looks fine of course, just revert these automatic changes if you please

Reply:
Sure thing, done.